### ISSUE TITLE
Problems with downDelta in KeyboardDragListener

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/scenery-phet#454]
- [x] downDelta should default to zero, most usages of KeyboardDragListener will not want this and will use smooth animated dragging without an initial movement on key press.
- [x] dragVelocity default is determined by this option, they should be independent.
- [x] the downDelta option moveOnHoldDelay, and moveOnHoldInterval options are mean to be used together to create a different drag style that moves the object every interval. This isn't clear from the documentation or organization of the options.
- [ ] In https://github.com/phetsims/friction/issues/166 we found that anything using the downDelta feature will suffer from an issue with JAWS because JAWS sends one keyUp event for EVERY keyDown event on `role="application"` elements. This is only true for the arrow keys. We submitted a bug report to VFO and they said they would fix this two years ago. Not sure whether we can accept this or need to design a workaround.

**Comment by jessegreenberg** [comment]
The final checkbox deserves its own issue. Closing this one, and Ill make a separate issue for that one.
